bedrock-devnet/devnet/__init__.py

```diff
@@ -127,6 +127,25 @@ def devnet_deploy(paths):
     devnet_cfg_orig = pjoin(paths.contracts_bedrock_dir, 'deploy-config', 'devnetL1.json')
     devnet_cfg_backup = pjoin(paths.devnet_dir, 'devnetL1.json.bak')
     shutil.copy(devnet_cfg_orig, devnet_cfg_backup)
+    deploy_config = read_json(devnet_cfg_orig)
+    l1BlockTag = l1BlockTagGet()["result"]
+    log.info(l1BlockTag)
```

Comment:
Should we remove this log since it seems useless for users?

Reply:
No, we need this log to know the situation of L1